streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -81,9 +81,40 @@ public void register(final StateRestorer restorer) {
 
         final Set<TopicPartition> restoringPartitions = new HashSet<>(needsRestoring.keySet());
         try {
-            final ConsumerRecords<byte[], byte[]> allRecords = restoreConsumer.poll(10);
+            final Set<ConsumerRecords<byte[], byte[]>> allRecords = new HashSet<>();
+            while (true) {
+                final ConsumerRecords<byte[], byte[]> records = restoreConsumer.poll(10);
```

Comment:
Just curious; why poll for 10ms in particular?

Reply:
When I looked through the code, it was set to 10 previously. I just set it to this value
to follow the previous versions as much as possible.